pyproject.toml

```diff
@@ -1,28 +1,28 @@
 [build-system]
-requires = ["setuptools>=62.0", "setuptools-git-versioning<2"]
+requires = ["setuptools>=62.0", "setuptools-git-versioning>=2.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "regolith"
 dynamic=['version', 'dependencies']
 authors = [
-  { name="Simon J.L. Billinge group", email="simon.billinge@gmail.com" },
+  { name="Simon J.L. Billinge group", email="sb2896@columbia.edu" },
   { name="Anthony Scopatz", email="scopatz@gmail.com" },
 ]
 maintainers = [
-  { name="Simon J.L. Billinge group", email="simon.billinge@gmail.com" },
+  { name="Simon J.L. Billinge group", email="sb2896@columbia.edu" },
   { name="Anthony Scopatz", email="scopatz@gmail.com" },
 ]
 description = "Python package for research group content management system"
-keywords = ["content management system"]
+keywords = ['content management system']
 readme = "README.rst"
 requires-python = ">=3.11, <3.14"
 classifiers = [
         'Development Status :: 5 - Production/Stable',
         'Environment :: Console',
         'Intended Audience :: Developers',
         'Intended Audience :: Science/Research',
-        'License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication',
+        'License :: OSI Approved :: BSD License',
```

Comment:
In general, don't EVER change the license (unless there is an error, which there isn't here) Which means we want to bring the old license text here.